packages/core/src/components/EditorSender/index.vue

```
@@ -456,6 +457,7 @@ onBeforeUnmount(() => {
 
 /** 暴露方法 */
 defineExpose({
+  submit:onSubmit(),
```

ESLint

[error] Missing space before value for key 'submit'.

(style/key-spacing)